apps/web/src/pages/Applications/ApplicationEducationPage/ApplicationEducationPage.tsx

```diff
@@ -349,22 +350,37 @@ const ApplicationEducation = ({
     },
     {
       value: EducationRequirementOption.Education,
-      label: intl.formatMessage({
-        defaultMessage: "I meet the 2-year post-secondary option",
-        id: "rGqohv",
-        description:
-          "Radio group option for education requirement filter in application education form.",
-      }),
+      label: isIAP
+        ? intl.formatMessage({
+            defaultMessage: "I have a high school diploma or GED equivalent.",
+            id: "u1Jsln",
+            description:
+              "Radio group option for education requirement filter in Indigenous apprenticeship application education form.",
+          })
+        : intl.formatMessage({
+            defaultMessage: "I meet the 2-year post-secondary option",
+            id: "rGqohv",
+            description:
+              "Radio group option for education requirement filter in application education form.",
+          }),
       contentBelow: (
         <div data-h2-margin="base(x.5, 0, x.5, x1)">
           <p>
-            {intl.formatMessage({
-              defaultMessage:
-                "Successful completion of two years of post secondary education in computer science, information technology, information management or another specialty relevant to this advertisement.",
-              id: "socp8t",
-              description:
-                "Message under radio button in application education page.",
-            })}
+            {isIAP
+              ? intl.formatMessage({
+                  defaultMessage:
+                    "Successful completion of a standard high school diploma or general education development (GED) equivalent.",
```

Comment:
General Education Development is a proper noun and should be title case.

Reply:
Updated as part of [fix typos, translation ids/descriptions](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6668/commits/8b3ae1fd9890817bbe29d235185b3fe9f0318389)